kuzzle-sdk-java/doc/3/core-classes/kuzzle/connect/index.md

```diff
@@ -0,0 +1,24 @@
+---
+code: true
+type: page
+title: connect
+description: Connects to Kuzzle using the `host` property provided in the [websocket parameters](/sdk/java/3/protocols/websocket/constructor#arguments).
+---
+
+# connect
+
+Connects to Kuzzle using the `host` property provided in the [websocket parameters](/sdk/java/3/protocols/websocket/constructor#arguments). Subsequent call have no effect if the SDK is already connected.
+
+## Arguments
+
+```java
+public void connect() throws Exception;
+```
+
+## Exceptions
+
+Throw an `Exception` when an error occur during the connection.
```

Comment:
```suggestion
Throws an `Exception` when an error occurs during the connection.
```

Also... wouldn't a [ConnectException](https://docs.oracle.com/en/java/javase/13/docs/api/java.base/java/net/ConnectException.html) be better suited?